build.rs

```diff
@@ -230,13 +230,29 @@ fn parse_header_defines(header_path: impl AsRef<Path>) -> Result<HashMap<String,
     Ok(definitions)
 }
 
-fn fix_config_map(mut config_map: HashMap<String, String>) -> HashMap<String, String> {
+fn fix_config_map(
+    mut config_map: HashMap<String, String>,
+    interpreter_config: &InterpreterConfig,
+) -> HashMap<String, String> {
+    if interpreter_config.version.major == 3 && interpreter_config.version.minor.unwrap_or(0) > 7 {
+        if let Some("1") = config_map.get("Py_DEBUG").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_REF_DEBUG".to_owned(), "1".to_owned());
+        }
+        if let Some("1") = config_map.get("Py_TRACE_REFS").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_TRACE_REFS".to_owned(), "1".to_owned());
+        }
+        // find CFLAGS to whether contains COUNT_ALLOCS then set it as 1
+        // if interpreter_config.version.minor.unwrap_or(0) < 9 {
+        // }
+        return config_map;
+    }
+
     if let Some("1") = config_map.get("Py_DEBUG").as_ref().map(|s| s.as_str()) {
         config_map.insert("Py_REF_DEBUG".to_owned(), "1".to_owned());
         config_map.insert("Py_TRACE_REFS".to_owned(), "1".to_owned());
-        config_map.insert("COUNT_ALLOCS".to_owned(), "1".to_owned());
+        // find CFLAGS to whether contains the COUNT_ALLOCS then set
+        // config_map.insert("COUNT_ALLOCS".to_owned(), "1".to_owned());
```

Comment:
I think this code might originally have been wrong (I can't see any suggestion that `Py_DEBUG` would set `COUNT_ALLOCS` in documentation), so please just delete:

```suggestion
```

Reply:
Yes,  I also said according the CFlags to decide whether to set COUNT_ALLOCS as 1.
But always we build python with pydebug, then  in this case , we then  can decide to enable COUNT_ALLOCS.  So i place the comment in the if Py_DEBUG block. 
Of course, if decide never support COUNT_ALLOCS( we know > 3.9 have remove this code), we just delete this code.